Description: Disable coveralls
 Disable coveralls during tests.
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/spec/spec_helper.rb
+++ b/spec/spec_helper.rb
@@ -1,7 +1,5 @@
 $:.unshift File.expand_path('..', __FILE__)
 $:.unshift File.expand_path('../../lib', __FILE__)
-require 'simplecov'
-SimpleCov.start
 require 'rspec'
 require 'rspec/its'
 require 'rack/test'
